src/io/jesi/backpack/test/runner.cljs

```diff
@@ -45,12 +55,13 @@
     (println stack)))
 
 (defn start []
+  (create-log-node)
```

Comment:
there was an issue with shadow-cljs where the log node didn't exist. We were then trying to append to nil...